brownie/network/transaction.py

```diff
@@ -163,6 +171,7 @@ def __init__(
         self.status = -1
         self.txid = txid
         self.fn_name = name
+        self.required_confs = required_confs
```

Comment:
Do you think this is important to be part of the public API? Given it only determines how long the initial call is blocking for, I can't really see why a user would want to reference it later.

Reply:
`tx.wait()` will set this attribute, so I thought it might be useful if you can call it before calling `tx.wait()`. But I can see moving this to an internal attribute